test/library/packages/SortedSet/general/kth/noDefaultValue.good

```diff
@@ -1,4 +1,4 @@
-$CHPL_HOME/modules/packages/SortedSet.chpl:275: In method 'kth':
-$CHPL_HOME/modules/packages/SortedSet.chpl:276: error: kth is not available on types that can't be 
+$CHPL_HOME/modules/packages/SortedSet.chpl:283: In method 'kth':
+$CHPL_HOME/modules/packages/SortedSet.chpl:284: error: kth is not available on types that can't be 
```

Comment:
might be worth adding the line number prediff to this test so it doesn't have to be updated again